fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js

```diff
@@ -1,51 +1,40 @@
 import { useState, useEffect } from "react";
 import { useAuth0 } from '@auth0/auth0-react';
-import configData from '../../config.json'
-import LoginButton from "../../components/authentication/login";
-import LogoutButton from "../../components/authentication/logout";
-import axios from 'axios';
 import NavNotLoggedIn from "../../components/navigation/NotLoggedIn/navNotLoggedIn";
-import FooterNotLoggedIn from "../../components/footer/footerNotLoggedIn/footerNotLoggedIn";
 import NavLoggedIn from "../../components/navigation/loggedIn/navLoggedIn";
 import { Link } from 'react-router-dom';
-import { Container, Row, Col, Modal, Button } from 'react-bootstrap';
 import './AdminAppointment.css';
-import { FaSearch } from 'react-icons/fa';
-
+import Filter from "../../components/AdminPanel/Filter";
+import { useGetAccessToken } from "../../components/authentication/authUtils";
 
 
 function AdminAccounts() {
 
     const {
         isAuthenticated,
-        getAccessTokenSilently,
     } = useAuth0();
 
     const [appointments, setAppointments] = useState([]);
     const [accessToken, setAccessToken] = useState(null);
+    const getAccessToken = useGetAccessToken();
+
+    const [searchTerm, setSearchTerm] = useState([["appointmentid",""], ["accountid",""], ["status",""]]);
+
+    const labels = ["Appointment ID", "Account ID", "Status"];
 
     useEffect(() => {
-        if (isAuthenticated) {
-            const getAccessToken = async () => {
-                try {
-                    const token = await getAccessTokenSilently({
```

Comment:
Is there a reason why we are not using getAccessTokenSilently here?

Reply:
This is getting the access token form the AuthUtils file it just take less space in the code when you need the access token in your views.